src/features/defi/providers/fox-eth-lp/hooks/useFoxEthLiquidityPool.ts

```diff
@@ -138,16 +121,17 @@ export const useFoxEthLiquidityPool = (
           bnOrZero(foxAmount).times(bn(10).exponentiatedBy(foxAsset.precision)).toFixed(0),
           calculateSlippageMargin(foxAmount, foxAsset.precision),
           calculateSlippageMargin(ethAmount, ethAsset.precision),
-          accountAddress,
+          fromAccountId(accountId).account,
           Date.now() + 1200000,
         ])
         const adapterType = adapter.getChainId()
+        const contractAddress = fromAssetId(uniswapV2Router02AssetId).assetReference
         const estimatedFees = await (adapter as unknown as EvmBaseAdapter<EvmChainId>).getFeeData({
```

Comment:
cast the adapter on line 77 to avoid this mess everywhere - we can also hardcode this to evm mainnet

Reply:
12a34820606b320dae5d4acf61f47f9f32aedaff